### PR TITLE
Wizards now struggle with tech like flashes and batons.

### PR DESCRIPTION
**Pull request by DimWhat**
[balance]

## About the PR 
This PR gives wizards a trait called "Technological Ineptitude" that makes them have the effects of clumsy on specifically batons and  flashes. I do have desire to make wizards also fumble with guns but it seems there is nothing really there for clumsy at the moment.
This change should make wizards less inclined to use security tools to become unstoppable and instead destroy security tools.

## Why's this needed? 
Wizard is currently extremely powerful when they get tasers and batons to an absurd degree since they can warp anywhere to charge them. The end result is not very fun to fight against as wizards usually just grab batons after using magic missile and steal them for their own use.

## Changelog

```changelog
(u)DimWhat
(*)Wizards now fumble with batons and flashes
```

### CODE REVIEW

File: code/obj/item/stun_baton.dm [hunk 1]
```diff
@@ -405,7 +405,7 @@
 		//never should happen but w/e
 
 		//make it harder for them clowns...
-		if (src.can_stun() == 1 && user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50))
+		if (src.can_stun() == 1 && user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50) || (user.traitHolder.hasTrait("tech_inept") && prob (50)))
```
Comment (Zonespace27) [inline code/obj/item/stun_baton.dm:408]:
```suggestion
		if (src.can_stun() && ((user.bioHolder?.HasEffect("clumsy") && prob(50)) || (user.traitHolder?.hasTrait("tech_inept") && prob(50))))
```

File: code/obj/item/stun_baton.dm [hunk 2]
```diff
@@ -242,7 +242,7 @@
 			user.show_message("<span class='alert'>[M] seems to be warded from attacks!</span>")
 			return
 
-		if (src.can_stun() == 1 && user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50))
+		if (src.can_stun() == 1 && user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50) || (user.traitHolder.hasTrait("tech_inept") && prob (50)))
```
Comment (Zonespace27) [inline code/obj/item/stun_baton.dm:245]:
```suggestion
		if (src.can_stun() && ((user.bioHolder?.HasEffect("clumsy") && prob(50)) || (user.traitHolder?.hasTrait("tech_inept") && prob(50))))
```

File: code/obj/item/stun_baton.dm [hunk 3]
```diff
@@ -218,7 +218,7 @@
 
 		src.is_active = !src.is_active
 
-		if (src.can_stun() == 1 && user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50))
+		if (src.can_stun() == 1 && user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50) || (user.traitHolder.hasTrait("tech_inept") && prob (50)))
```
Comment (Zonespace27) [inline code/obj/item/stun_baton.dm:221]:
```suggestion
		if (src.can_stun() && ((user.bioHolder?.HasEffect("clumsy") && prob(50)) || (user.traitHolder?.hasTrait("tech_inept") && prob(50))))
```

File: code/obj/item/device/flash.dm [hunk 4]
```diff
@@ -208,7 +208,7 @@
 	if (src.l_time && world.time < src.l_time + 10)
 		return
 
-	if (user?.bioHolder?.HasEffect("clumsy") && prob(50))
+	if (user?.bioHolder?.HasEffect("clumsy") && prob(50) || (user.traitHolder.hasTrait("tech_inept") && prob (50)))
```
Comment (Zonespace27) [inline code/obj/item/device/flash.dm:211]:
```suggestion
	if ((user?.bioHolder?.HasEffect("clumsy") && prob(50)) || (user?.traitHolder?.hasTrait("tech_inept") && prob(50)))
```

File: code/obj/item/device/flash.dm [hunk 5]
```diff
@@ -95,7 +95,7 @@
 	if (t.loc:sanctuary)
 		user.visible_message("<span class='alert'><b>[user]</b> tries to use [src], cannot quite comprehend the forces at play!</span>")
 		return
-	if (user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50))
+	if (user.bioHolder && user.bioHolder.HasEffect("clumsy") && prob(50) || (user.traitHolder.hasTrait("tech_inept") && prob (50)))
```
Comment (Zonespace27) [inline code/obj/item/device/flash.dm:98]:
```suggestion
	if ((user.bioHolder?.HasEffect("clumsy") && prob(50)) || (user.traitHolder?.hasTrait("tech_inept") && prob(50)))
```

**Comment by flappybatpal** [comment]
Good idea, I'd be interested to see what can be done with guns.

**Comment by DimWhat** [comment]
Reopening cause i'm feeling like this should probably get worked on again.

**Pull request closed without merging**